### PR TITLE
Introduce LongBinaryPacking

### PR DESCRIPTION
**Pull request by blacelle**

### CODE REVIEW

**Comment by blacelle** [comment]
@lemire This is ready for review

File: src/main/java/me/lemire/longcompression/LongUtil.java [hunk 1]
```diff
@@ -15,7 +15,37 @@
  */
 @Deprecated
 public class LongUtil {
+	
+	/**
```
Comment (lemire) [inline src/main/java/me/lemire/longcompression/LongUtil.java:19]: Extra space before the '/**'.

**Comment by ethlo** [comment]
Hi! This looks like what I need for my project. Is there a chance that this can be merged (and hopefully released on Maven central? Is there anything I can contribute to make this happen? Thank you for your time.

**Comment by blacelle** [comment]
I resynced the PR

**Comment by lemire** [comment]
Thanks. Merging.

**Comment by ethlo** [comment]
Hi again! Thank you for the quick response. I built the latest snapshot and the new code is working, but I was a bit surprised that the encoding would be that much less efficient than the integer encoder for the same input of numbers. I'm unfortunately not well-versed in the integer encoding schemes to know if this is as expected. 

Input is 500,000 integer values between 0 and 1,500,000.

`IntegratedIntCompressor`
Delta: 269 KB

`LongCompressor`
Delta: 2554 KB

Any input greatly appreciated!

**Comment by blacelle** [comment]
`IntegratedIntCompressor` is dedicated to sorted integers, while `LongCompressor` manages unsorted integers. There is no `IntegratedLongCompressor` yet.

**Pull request merged**